scripts/change_url

```diff
@@ -75,35 +81,40 @@ fi
 if [ $change_domain -eq 1 ]
 then
 	# Delete file checksum for the old conf file location
-	ynh_delete_file_checksum "$nginx_conf_path"
+	ynh_delete_file_checksum --file="$nginx_conf_path"
 	mv $nginx_conf_path /etc/nginx/conf.d/$new_domain.d/$app.conf
 	# Store file checksum for the new config file location
-	ynh_store_file_checksum "/etc/nginx/conf.d/$new_domain.d/$app.conf"
+	ynh_store_file_checksum --file="/etc/nginx/conf.d/$new_domain.d/$app.conf"
 fi
 
 #=================================================
 # SPECIFIC MODIFICATIONS
 #=================================================
 # MODIFY URL IN .ENV
 #=================================================
-ynh_print_info "Updating .env configuration..."
+ynh_print_info --message="Updating .env configuration..."
 
-ynh_systemd_action --action=stop --service_name=$app --log_path="systemd"
-ynh_replace_string "$old_domain$old_path" "$new_domain$new_path" "$final_path/.env"
-ynh_systemd_action --action=start --service_name=$app --log_path="systemd" --line_match="Kadira: completed instrumenting the app"
-sleep 10
+ynh_replace_string --match_string="$old_domain$old_path" --replace_string="$new_domain$new_path" --target_file="$final_path/.env"
 
 #=================================================
 # GENERIC FINALISATION
+#=================================================
+# START SYSTEMD SERVICE
+#=================================================
+ynh_print_info --message="Starting a systemd service..."
```

Comment:
```suggestion
ynh_print_info --message="Starting $app service..."
```

Reply:
Bump for this last one :P (I think it's hidden in the github list)

Reply:
Yep, it was hidden ^^